### PR TITLE
Add Rosie's photo and card

### PR DESCRIPTION
**Pull request by Rosie-m**
image

### CODE REVIEW

**Review by jaywonchung — CHANGES_REQUESTED** [review]
Thank you for the PR! Could you compress the image so that it's around or under 50 KB? Just so that the website loads quickly.

File: index.html [hunk 1]
```diff
@@ -326,6 +326,20 @@ <h6>Ph.D. Student</h6>
...
                 <!-- END SINGLE TEAM -->
                 <!-- START SINGLE TEAM -->
+                <div class="col-md-4 col-sm-6">
+                    <div class="single-team text-center">
+                        <img src="assets/img/members/luoxi.jpg" style="max-width: 83%" class="center-block img-circle" />
+                        <h4 class="team-member">Luoxi Meng</h4>
+                        <h6>MS Student</h6>
+                        <p><a href="https://symbioticlab.org">SymbioticLab</a>, University of Michigan</p>
+                        <ul class="team-social">
+                            <li><a href="https://github.com/Rosie-m"><i class="fa fa-home"></i></a>
```
Comment (jaywonchung) [inline index.html]:
```suggestion
                            <li><a href="https://github.com/Rosie-m"><i class="fa fa-github"></i></a>
```

Comment (Rosie-m) [reply index.html]: Thanks for fixing it!

**Review by jaywonchung — APPROVED** [review]
LGTM!

Just as an FYI, you can directly apply a suggestion from the comment by clicking "Commit suggestion" in the web UI.

**Pull request merged**